examples/wistiaplayer.amp.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="amps.html">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+    <script async custom-element="amp-wistia-player" data-amp-report-test="amp-wistia-player" src="https://cdn.ampproject.org/v0/amp-wistia-player-0.1.js"></script>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+  </head>
+  <body>
+    <amp-wistia-player
+      height=360
+      width=512
+      data-media-id="u8p9wq6mq8">
```

Comment:
What do you think about changing this to `data-hashed-id`? I realize it's not super semantic, but it's the term we use in our docs. cc @mrdavidjcole for thought.

Reply:
Hm - we have multiple types of hashed_ids now (due to API v2) - how about `media-hashed-id`?

Reply:
`media-hashed-id` makes sense to me.

Reply:
Not sure if this is obvious, but I think it still needs to start with `data-` to be valid markup--that's why all the others start with `data-` too.

Reply:
Yup - it would have to be `data-media-hashed-id`